chef/automate
# Event feed project update

### :nut_and_bolt: Description: What code changed, and why?
Adding project update to the event-feed. This will only update the events from Chef Server Actions. This adds authz-service as a dependence. 

### :chains: Related Resources

#444 

### :athletic_shoe: How to Build and Test the Change
1. `build components/event-feed-service && build components/automate-deployment && build components/authz-service && start_all_services`
1. Add three chef-server events with different values organization values ("org1", "org2", and "org3"). Do not assign any projects. 
1. `generate_random_chef_server_event | jq '.data.chef_organization = "org1" | del(.Projects)' | send_event`
1. `generate_random_chef_server_event | jq '.data.chef_organization = "org2" | del(.Projects)' | send_event`
1. `generate_random_chef_server_event | jq '.data.chef_organization = "org3" | del(.Projects)' | send_event`
1. Create a project named "project1" with a rule of type "event" that matches "chef organization" with "org1"
1. Create a project named "project2" with a rule of type "event" that matches "chef organization" with "org2"
1. Run the project update
1. After the update is complete, open the event feed page. https://a2-dev.test/dashboards/event-feed.
1. Filter on project1 and ensure only one event is displayed.
1. Filter on project2 and ensure only one event is displayed.
1. Remove all project filters and ensure all three events are displayed.  

### :white_check_mark: Checklist
